tests/components/figure.test.ts

```diff
@@ -221,6 +221,22 @@ describe("figures", () => {
     expect(model.getters.getFigure(sheetId, figureId)).toMatchObject(expectedSize);
   });
 
+  test.each([
+    ["top", { mouseOffsetX: 0, mouseOffsetY: -100 }, { width: 100, height: 100 }],
+    ["left", { mouseOffsetX: -100, mouseOffsetY: 0 }, { width: 100, height: 100 }],
+    ["topLeft", { mouseOffsetX: -100, mouseOffsetY: -100 }, { width: 100, height: 100 }],
+  ])(
+    "Resizing a figure through its top and left anchor does not change size beyond header boundaries",
+    async (anchor: string, mouseMove, expectedSize) => {
```

Comment:
expectedSize is kinda useless.

nitpick: either type all the arguments, or let typescript deduce the type for all arguments, do'nt type half of them